lib/stripe/stripe_client.rb

```diff
@@ -7,21 +7,29 @@ module Stripe
   # recover both a resource a call returns as well as a response object that
   # contains information on the HTTP call.
   class StripeClient
+    include Stripe::ClientAPIOperations
+
     # A set of all known thread contexts across all threads and a mutex to
     # synchronize global access to them.
     @thread_contexts_with_connection_managers = []
     @thread_contexts_with_connection_managers_mutex = Mutex.new
     @last_connection_manager_gc = Util.monotonic_time
 
-    # Initializes a new `StripeClient`.
-    #
-    # Takes a connection manager object for backwards compatibility only, and
-    # that use is DEPRECATED.
-    def initialize(_connection_manager = nil)
+    # Initializes a new StripeClient
+    def initialize(config_overides = {})
       @system_profiler = SystemProfiler.new
       @last_request_metrics = nil
+      @config_overides = config_overides
```

Comment:
In e.g. stripe-php we support two different syntaxes for initializing clients:

1. passing a configuration hash (what you're doing already)
2. passing just the API key as a string

Would you mind adding support for the latter? I think it should be as simple as:
```ruby
@config_overides = config_overrides.is_a?(String) ? {api_key: config_overrides} : config_overrides
```

Reply:
That's a great suggestion and certainly improves usability. I'll get it added!